src/align.cpp

```diff
@@ -134,15 +156,23 @@ int main(int argc, char** argv) {
 
   // downsampling
   pcl::ApproximateVoxelGrid<pcl::PointXYZ> voxelgrid;
-  voxelgrid.setLeafSize(0.1f, 0.1f, 0.1f);
+  constexpr float leaf_size = 0.1;
+  voxelgrid.setLeafSize(leaf_size, leaf_size, leaf_size);
+  pcl::CropBox<pcl::PointXYZ> crop_box;
```

Comment:
Because it causes an error on the example data, could you remove this crop_box or change its cropping range?

Reply:
oops, these changes shouldnt go in this PR actually, same with the visualization